group_project/assignment_2.py

```diff
@@ -1,6 +1,14 @@
 def all_in_string(a_string, s1, s2, s3):
-    pass
-
+    if s1 in a_string:
+        a = 1
+    else: a = 0
+    if s2 in a_string:
+        b = 1
+    else: b = 0
+    if s3 in a_string:
+        c = 1
+    else: c = 0
+    return a + b + c
 
```

Comment:
My code was very similar to yours. 
Instead of assigning the letters a, b, c, with the number 1, I created an empty string and I added numbers to it: 
new_string = 0
if s1 in a_string:
    new_string = new_string + 1
if s2 in a_string:
    new_string = new_string + 1
if s3 in a_string:
    new_string =  new_string + 1
return new_string